bbc/simorgh
# WSTEAM1-245 AMP: Cookie banner needs to meet a11y 'focus not obscured' guideline

Resolves JIRA [WSTEAM1-245](https://jira.dev.bbc.co.uk/browse/WSTEAM1-245)

Overall changes
======
Adds a close button to AMP cookie and privacy banners

Code changes
======

- Adds an 'X' icon
- Adds a close button in  AmpCookieBanner and psammead consent banner
- Adds new 'focusIndicatorInset' global style

Testing
======
1.  Pull down this branch, run yarn dev and navigate to http://localhost:7080/portuguese.amp
2. Confirm the close button is there, styled correctly and has the expected behaviour as per the AAC guidelines

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Designs](https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/63be9d6a7d7c960d0a8c9a35)

[AAC](https://paper.dropbox.com/doc/Cookie-banner-Privacy-banner-AAC-Close-button--CCM0ApcQaem50uf8vjxOwMgVAg-lRrcILmSEmB4OQjHaiYGb)

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
